lib/searchable_by.rb

```diff
@@ -45,8 +45,6 @@ def self.norm_values(query)
       values = []
       query  = query.to_s.dup
       query.gsub!(/(\-|\+?)"([^"]+)"/) {|_| values.push("#{Regexp.last_match(1)}#{Regexp.last_match(2)}"); '' }
```

Comment:
to fix the issue below, you could try `"([^"]*)"` instead of `"([^"]+)"`

Reply:
This does solve the issue in the above comment but causes the test       
`expect(norm('-+""')).to eq(%w[-+""])` to fail and get (['+', '-']) which is why I had the line `query.gsub!(/(")/, '')` before as it allowed all tests to pass.
What behaviour would you like for this case?

Reply:
instead of `(\-|\+?)` -> `([\-\+]?)`